package.json

```diff
@@ -27,5 +27,8 @@
     "nohoist": [
       "**/jsii-calc-bundled"
     ]
+  },
+  "dependencies": {
+    "yarn": "^1.19.1"
```

Comment:
I would rather not...

Reply:
This was actually added by accident, so I'm happy to take it out. But I do wonder: why not, actually? Isn't it a convenient way to bootstrap yarn by using NPM? And then we can keep it nice & isolated.